contracts/pallet-ibc/src/ics20_fee/mod.rs

```diff
@@ -81,6 +92,19 @@ pub mod pallet {
 	}
 }
 
+pub trait FlatFeeConverter {
+	type AssetId;
```

Comment:
Is there an issue adding this trait to composable traits? Although, it sort of makes sense to define FlatFeeConverter trait here because it is going to be used here